### PR TITLE
Create NJ yamls; edit and move other yamls

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/CAL000316.yaml [hunk 1]
```diff
@@ -37,8 +37,8 @@ contact_form:
           required: true
     - javascript:
         - name: accent-replace
-          values: ["$NAME_FIRST", "$NAME_LAST"]
-          selectors: ["#txtFirstName", "#txtLastName"]
+          values: ["$NAME_FIRST", "$NAME_LAST", "$ADDRESS_STREET"]
+          selectors: ["#txtFirstName", "#txtLastName", "#txtAddress"]
           commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/[èēéë]/g,'e').replace(/[àåáä]/g, 'a').replace(/[Á]/g, 'A').replace(/[.&,]/g, '').replace(/[ý]/g, 'y').replace(/[íï]/g, 'i').replace(/[óö]/g, 'o').replace(/[ñ]/g, 'n'); }"]
```
Comment (tirish) [inline members/CAL000316.yaml]: Do you need to add `.replace(/[Ó]/g, 'O')` to this one too?

Comment (OdeclasV) [reply members/CAL000316.yaml]: Not necessary for this one

File: members/B001135.yaml [hunk 2]
```diff
@@ -256,6 +255,111 @@ contact_form:
           required: true
           options:
             NC: "NC"
+        - name: county
+          selector: select#input-E649CFAD-5056-A066-604A-057270612948
+          value: $ADDRES_COUNTY
```
Comment (vramoscoreas) [inline members/B001135.yaml]: `ADDRESS`?

Comment (OdeclasV) [reply members/B001135.yaml]: blah

Comment (vramoscoreas) [reply members/B001135.yaml]: Sorry should it be `ADDRESS_COUNTY` instead of `ADDRES_COUNTY`?

**Comment by OdeclasV** [comment]
https://www.youtube.com/watch?v=k1CEolFz5do

**Pull request merged**